php/classes/integrations/elementor/class-template-importer.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace SeriouslySimplePodcasting\Integrations\Elementor;
+
+use Elementor\TemplateLibrary\Manager;
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Template Importer for Elementor
+ *
+ * @author      Jonathan Bossenger
+ * @category    Class
+ * @package     SeriouslySimplePodcasting/Classes
+ * @since       1.19.18
+ */
+class Template_Importer {
+
+	protected $template_name = 'ssp-elementor-template-2020-09-30.json';
+	protected $template_path = SSP_PLUGIN_PATH . 'templates/elementor/';
+
+	protected $template_library_manager;
+
+	public function __construct() {
+		$this->template_library_manager = new Manager();
+		$this->init();
+	}
+
+	public function init() {
+		add_action( 'init', array( $this, 'process_template_import' ) );
+	}
+
+	public function process_template_import() {
+		// @todo add is_admin checking
+		if ( ! is_admin() ) {
+			return;
+		}
+		// @todo add user caps checks
```

Comment:
@daniloradovic we need to add security checks to this process

1. We should pass a nonce in this request, and verify it to be sure it's the same as from the url being passed. -> https://developer.wordpress.org/reference/functions/wp_verify_nonce/
2. Do we have a valid logged in user, and does that user have access to make this request. See this useful article for the correct checks to make -> https://dev.to/lucagrandicelli/why-isadmin-is-totally-unsafe-for-your-wordpress-development-1le1